wallet/how-to/connect/set-up-sdk/javascript/react/index.md

```diff
@@ -0,0 +1,153 @@
+---
+sidebar_label: React
+sidebar_position: 1
+---
+
+# Use MetaMask SDK with React
+
+You can import [MetaMask SDK](../../../../../concepts/sdk.md) into your React dapp to enable your users to
+easily connect to the MetaMask browser extension and MetaMask Mobile.
+The SDK for React has the [same prerequisites](../index.md#prerequisites) as for standard JavaScript.
+
+:::info React UI
+This page provides instructions for using the standard `@metamask/sdk-react` package.
+Alternatively, you can use the [`@metamask/sdk-react-ui`](react-ui.md) package to easily use
+[wagmi](https://wagmi.sh/) hooks and a pre-styled UI button component for connecting to MetaMask.
+:::
+
+:::tip Examples
+Refer to the [MetaMask JavaScript SDK examples](https://github.com/MetaMask/metamask-sdk/tree/main/packages/examples)
+for advanced use cases.
+:::
+
+## Steps
+
+### 1. Install the SDK
+
+In your project directory, install the SDK using Yarn or npm:
+
+```bash
+yarn add @metamask/sdk-react
+```
+
+or
+
+```bash
+npm i @metamask/sdk-react
+```
+
+### 2. Import the SDK
+
+In your project script, add the following to import the SDK:
+
+```javascript
+import { MetaMaskProvider } from '@metamask/sdk-react';
+```
+
+### 3. Wrap your project with MetaMaskProvider
```

Comment:
Can we format `MetaMaskProvider`? You may have mentioned there's an issue with using formatting in headings.